### PR TITLE
added js question by atahan

### PR DESCRIPTION
**Pull request by AtahanKocc**
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] New Question
- [ ] Revision of an existing question
- [ ] Infrastructure change (automation, etc.)
- [ ] Other (please elaborate) 

# Checklist:

- [ ] My content follows the style guidelines of this project
- [ ] I have performed a self-review of my own content

Pull requests should be thought of as a conversation. There will be some back and forth when trying to get code merged into this or any other project. With all but the simplest changes you can and should expect that the maintainers of the project will request changes to your code. Please be aware of that and check in after you open your PR in order to get your code merged in cleanly. 

Thanks!

### CODE REVIEW

File: src/questions/javascript-questions.md [hunk 1]
```diff
@@ -52,6 +52,9 @@ permalink: /questions/javascript-questions/index.html
 * What is the difference between `while` and `do-while` loops in JavaScript?
 * What is a promise? Where and how would you use promise?
 * Discuss how you might use Object Oriented Programming principles when coding with JavaScript.
+* What are all the types of Pop up boxes available in JavaScript?
+* Why do we use the word “debugger” in javascript?
+* What are object prototypes?
```
Comment (veronvynguyen) [inline src/questions/javascript-questions.md:57]: this question may be duplicate of 
`Explain how prototypal inheritance works.`
?

File: src/questions/javascript-questions.md [hunk 2]
```diff
@@ -52,6 +52,9 @@ permalink: /questions/javascript-questions/index.html
 * What is the difference between `while` and `do-while` loops in JavaScript?
 * What is a promise? Where and how would you use promise?
 * Discuss how you might use Object Oriented Programming principles when coding with JavaScript.
+* What are all the types of Pop up boxes available in JavaScript?
+* Why do we use the word “debugger” in javascript?
```
Comment (veronvynguyen) [inline src/questions/javascript-questions.md:56]: what is the question testing? knowledge of what debugger does? 

might be better to rephrase 
`Explain what the debugger key does.`

**Review by veronvynguyen — CHANGES_REQUESTED** [review]

File: src/questions/javascript-questions.md [hunk 3]
```diff
@@ -52,6 +52,9 @@ permalink: /questions/javascript-questions/index.html
 * What is the difference between `while` and `do-while` loops in JavaScript?
 * What is a promise? Where and how would you use promise?
 * Discuss how you might use Object Oriented Programming principles when coding with JavaScript.
+* What are all the types of Pop up boxes available in JavaScript?
```
Comment (roblarsen) [inline src/questions/javascript-questions.md:55]: I'm not sure what this is asking

File: src/questions/javascript-questions.md (hunk 2 above)
Comment (roblarsen) [reply src/questions/javascript-questions.md:56]: Hopefully you've pre-screened someone who doesn't know what debugging is.

File: src/questions/javascript-questions.md (hunk 1 above)
Comment (roblarsen) [reply src/questions/javascript-questions.md:57]: This is a dupe

**Comment by roblarsen** [comment]
Thanks for your contribution. I don't think these make sense for the project at this time.

**Pull request closed without merging**